### PR TITLE
Npm script seed for testing our application

### PR DESCRIPTION
**Pull request by sonakollarova5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,6 +64,7 @@
     "stylelint-scss": "^3.3.1",
     "ts-node": "~7.0.0",
     "tslint": "~5.11.0",
-    "typescript": "~2.9.2"
+    "typescript": "~2.9.2",
+    "typescript-require": "^0.2.10"
```
Comment (Ency88) [inline package.json]: we don't need `typescript-require` we should use .ts file for run and use `ts-node` to run it

File: fill-test-environment.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+firebase.auth().createUserWithEmailAndPassword(production.testUser2.email, production.testUser2.password)
+  .then(user => console.log(user))
+  .catch(error => console.error(error));
+
+firebase.auth().createUserWithEmailAndPassword(production.testUser3.email, production.testUser2.password)
+  .then(user => console.log(user))
+  .catch(error => console.error(error));
+
+//TODO fill 10 working days for each user (at least one break during day)
```
Comment (Ency88) [inline fill-test-environment.js]: missing `TODO`

File: fill-test-environment.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const firebase = require("firebase");
```
Comment (Ency88) [inline fill-test-environment.js]: this file should be `fill-test-environment.ts`

File: production.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  password: 'user123',
+};
+
+export const testUser3 = {
+  email: 'test@senacor.com',
+  password: 'test',
+};
+
+export const workingDay1 = {
+  from: 'October 30, 2018 7:30:00 AM',
```
Comment (Ency88) [inline production.ts]: how firebase store this information? as `string` or as `date`. 
here we should use the same interfaces which application is using. As far this is part of this sprint, there should be comunication with the next members

File: package.json (hunk 1 above)
Comment (sonakollarova5) [reply package.json]: typescript-require was removed 💯

File: fill-test-environment.js (hunk 2 above)
Comment (sonakollarova5) [reply fill-test-environment.js]: I've added 10 working days demo.

File: fill-test-environment.js (hunk 3 above)
Comment (sonakollarova5) [reply fill-test-environment.js]: File format changed.

File: production.ts (hunk 4 above)
Comment (sonakollarova5) [reply production.ts]: As a Date. Now it's using format of Majo's model entities.

**Pull request merged**